src/main/java/org/openrewrite/java/testing/junit4toassertj/ChangeAssertTrue.java

```diff
@@ -0,0 +1,124 @@
+package org.openrewrite.java.testing.junit4toassertj;
+
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.*;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.openrewrite.Formatting.EMPTY;
+import static org.openrewrite.Formatting.format;
+import static org.openrewrite.Tree.randomId;
+
+/**
+ * This is a refactoring visitor that will convert JUnit-style assertTrue() to assertJ asserts:
+ *
+ * assertTrue("This should be true", EXPRESSION) -> assert(EXPRESSION).with("This should be true").isTrue()
+ * assertTrue(EXPRESSION) -> assert(EXPRESSION).isTrue()
+ *
+ */
+public class ChangeAssertTrue extends JavaIsoRefactorVisitor {
```

Comment:
Since this visitor requires no configuration add `@AutoConfigure` to make it easily addressable from YAML

Reply:
This is so this visitor can be included declaratively in a recipe? I will add it.